src/controllers/livetv/livetvsuggested.js

```diff
@@ -1,395 +1,403 @@
-            var previousTabController = tabControllers[parseInt(evt.detail.previousIndex)];
+        loadTab(view, parseInt(evt.detail.selectedTabIndex));
+    }
 
-            if (previousTabController && previousTabController.onHide) {
-                previousTabController.onHide();
-            }
+    function getTabContainers() {
+        return view.querySelectorAll('.pageTabContent');
+    }
 
-            loadTab(view, parseInt(evt.detail.selectedTabIndex));
-        }
+    function initTabs() {
+        mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
+    }
 
-        function getTabContainers() {
-            return view.querySelectorAll('.pageTabContent');
-        }
+    function getTabController(page, index, callback) {
+        let depends;
 
-        function initTabs() {
-            mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
-        }
+        // TODO int is a little hard to read
+        switch (index) {
+            case 0:
+                depends = 'controllers/livetv/livetvsuggested';
+                break;
 
-        function getTabController(page, index, callback) {
-            var depends = [];
+            case 1:
+                depends = 'controllers/livetv/livetvguide';
+                break;
 
-            // TODO int is a little hard to read
-            switch (index) {
-                case 0:
-                    break;
+            case 2:
+                depends = 'controllers/livetv/livetvchannels';
+                break;
 
-                case 1:
-                    depends.push('controllers/livetv/livetvguide');
-                    break;
+            case 3:
+                depends = 'controllers/livetv/livetvrecordings';
+                break;
 
-                case 2:
-                    depends.push('controllers/livetv/livetvchannels');
-                    break;
+            case 4:
+                depends = 'controllers/livetv/livetvschedule';
+                break;
 
-                case 3:
-                    depends.push('controllers/livetv/livetvrecordings');
-                    break;
+            case 5:
+                depends = 'controllers/livetv/livetvseriestimers';
+                break;
 
-                case 4:
-                    depends.push('controllers/livetv/livetvschedule');
-                    break;
+            case 6:
+                depends = 'scripts/searchtab';
+        }
 
-                case 5:
-                    depends.push('controllers/livetv/livetvseriestimers');
-                    break;
+        import(depends).then(({default: controllerFactory}) => {
+            let tabContent;
 
-                case 6:
-                    depends.push('scripts/searchtab');
+            if (index === 0) {
+                tabContent = view.querySelector('.pageTabContent[data-index=\'' + index + '\']');
```

Comment:
```suggestion
                tabContent = view.querySelector(`.pageTabContent[data-index="${index}"]`);
```

Prefer template literals to string concatenation, it's nicer to read :)